packages/camera/camera_android_camerax/README.md

```diff
@@ -2,11 +2,12 @@
 
 An Android implementation of [`camera`][1] that uses the [CameraX library][2].
 
-*Note*: This package is under development, so please note the
-[missing features and limitations](#limitations), but
-otherwise feel free to try out the current implementation and provide any
-feedback by filing issues under [`flutter/flutter`][5] with `[camerax]` in
-the title, which will be actively triaged.
+*Note*: This implementation will eventually become the default implementation
```

Comment:
small nit: should we be more prescriptive of when we estimate this becomes the default? eventually makes it still seem like it is far off.

Reply:
Changed this to "by May" to give us until I/O. LMK what you think!